ops/terraform/services/pipeline/modules/bfd_pipeline_slis/lambda_src/update_pipeline_slis.py

```diff
@@ -59,7 +59,7 @@ class S3EventType(str, Enum):
     @classmethod
     def from_event_name(cls, event_name: str) -> "S3EventType":
         try:
-            return next(x for x in S3EventType if event_name in x)
+            return next(x for x in S3EventType if x in event_name)
```

Comment:
This is the only functional change; everything else is formatting changes or changes to tests.